cli/src/hooks/init/setup-channel.ts

```diff
@@ -0,0 +1,21 @@
+import { type Hook } from "@oclif/core";
+import { mkdir, writeFile } from "fs/promises";
+import { join } from "path";
+import { homedir } from "os";
+
+const hook: Hook<"init"> = async function () {
+	try {
+		const starkitDir = join(homedir(), ".local", "share", "starkit");
```

Comment:
Will this work on windows?

Reply:
I think we should reuse this logic or get it somehow - https://github.com/oclif/config/blob/7e9b7960e39948e462038287fca45deec2f86050/src/config.ts#L448